docker-compose.yml

```
@@ -47,6 +47,13 @@ services:
     ports:
       - "21:21"
       - "${FTP_MIN_PORT}-${FTP_MAX_PORT}:${FTP_MIN_PORT}-${FTP_MAX_PORT}"
+  pg:
+    container_name: pg
+    image: postgres:14.3
```

```suggestion
    image: postgres:14.3-alpine
```
I think it's better to use alpine when possible as they are smaller ([85 MB vs 133 MB](https://hub.docker.com/_/postgres/?tab=tags&page=1&name=14.3)) and more secure

---

Great point @inghylt! I should have been more careful about this when selecting the image. I tried this now and it seems to work fine. Have added a commit for it now.